src/pages/landing/HeroSection.tsx

```diff
@@ -1,15 +1,23 @@
 import styled from 'styled-components';
 
-import { Section, MOBILE_MAX_WIDTH, AnimatedTitle } from '~/components';
-import VIDEO_CHROME from '~/assets/videos/landing.webm';
-import VIDEO_SAFARI from '~/assets/videos/landing.mp4';
+import { Section, MOBILE_MAX_WIDTH, SquigglyTitle } from '~/components';
 
 const HeroSection = ({ ...props }) => {
   return (
     <>
       <StyledHeroSection full backgroundImage='/img/hero/hero-bg.jpg' {...props}>
         <TitleContainer>
-          <AnimatedTitle chromeSrc={VIDEO_CHROME} safariSrc={VIDEO_SAFARI} />
+          <SquigglyTitle
+            text={`THE LARGEST CORE DEVELOPMENT\nGROUP IN WEB3`}
```

Comment:
```suggestion
            text="THE LARGEST CORE DEVELOPMENT\nGROUP IN WEB3"
```

Reply:
```suggestion
            text={`THE LARGEST CORE DEVELOPMENT<br/>GROUP IN WEB3`}
```

Template string is necessary for breaking lines using `\n`.

We can use `<br />` instead and `dangerouslySetInnerHTML` into SquigglyTitle component.

Wdyt?

Reply:
> Template string is necessary for breaking lines using \n.
got it. so leave it as it is 😁